# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/perl/hotfix-x-bigendian.diff
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- perl-5.36.0/config.h	2022-10-25 10:41:08.707005658 +0200
+++ perl-5.36.0/config.h	2022-10-25 01:02:00.964000000 +0200
@@ -849,7 +849,7 @@
 #    endif
 #  endif
 #else
-#define BYTEORDER 0x12345678	/* large digits for MSB */
+#define BYTEORDER 0x4321	/* large digits for MSB */
 #endif
 
 /* CHARBITS:
@@ -3844,10 +3845,10 @@
  *	This symbol, if defined, is a comma-separated list of
  *	hexadecimal bytes (0xHH) for the long double precision not-a-number.
  */
-#define DOUBLEINFBYTES  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xf0, 0x7f		/**/
-#define DOUBLENANBYTES  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xf8, 0xff		/**/
-#define LONGDBLINFBYTES 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x80, 0xff, 0x7f, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00		/**/
-#define LONGDBLNANBYTES 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xc0, 0xff, 0xff, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00		/**/
+#define DOUBLEINFBYTES  0x7f, 0xf0, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00		/**/
+#define DOUBLENANBYTES  0x7f, 0xf8, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00		/**/
+#define LONGDBLINFBYTES 0x7f, 0xf0, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00		/**/
+#define LONGDBLNANBYTES 0x7f, 0xf8, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00		/**/
 
 /* PERL_PRIfldbl:
  *	This symbol, if defined, contains the string used by stdio to
